web/src/pages/starter-kits/general.tsx

```diff
@@ -0,0 +1,158 @@
+            <Heading level={1} styleVariant="h1Large" className="display-only-desktop">
+              {Config.starterKits.hero.title}
+              <br />
+            </Heading>
+            <Heading level={1} className="display-none-desktop">
+              {Config.starterKits.hero.title}
+              <br />
+            </Heading>
+            <div className="text-base-darkest font-sans-md margin-bottom-3">
+              {Config.starterKits.hero.subtitle}
+            </div>
+            <PrimaryButton
+              isColor={"secondary-vivid-dark"}
+              onClick={() => {
+                analytics.event.starter_kit_landing.click.start_now_button();
+                router.push(ROUTES.onboarding);
+              }}
+              isRightMarginRemoved={true}
+              isLargeButton={true}
+            >
+              {Config.starterKits.hero.ctaButton}
+            </PrimaryButton>
+          </div>
+
+          <div className="desktop:grid-col-6 desktop:display-block display-none">
+            <img className="width-100" src="/img/seo-img-main.webp" alt="" role="presentation" />
+          </div>
+        </section>
+        <section
+          aria-label="Solutions"
+          className={
+            "display-flex flex-column gap-5 padding-y-5 desktop:padding-y-10 padding-x-2 desktop:padding-x-0 grid-wrapper"
+          }
+        >
+          <div className="title-section">
+            <Heading level={2}>{Config.starterKits.solutions.title}</Heading>
+            <p className="font-sans-md">{Config.starterKits.solutions.subtitle}</p>
+          </div>
+          <div className="display-flex flex-column gap-2 cards-section">
+            <Card
+              iconFilename="check-circle-icon-blue"
+              title={Config.starterKits.solutions.card1.title}
+              description={Config.starterKits.solutions.card1.body}
+            />
+            <Card
+              iconFilename="dollar-circle-icon-purple"
+              title={Config.starterKits.solutions.card2.title}
+              description={Config.starterKits.solutions.card2.body}
+            />
+            <Card
+              iconFilename="id-badge-circle-icon-green"
+              title={Config.starterKits.solutions.card3.title}
+              description={Config.starterKits.solutions.card3.body}
+            />
+          </div>
+          <div className="cta-button">
+            <PrimaryButton
+              isColor="primary"
+              onClick={() => {
+                analytics.event.starter_kit_landing.click.get_my_starter_kit_button();
+                router.push(ROUTES.onboarding);
+              }}
+              isRightMarginRemoved={true}
+              isLargeButton={true}
+            >
+              {Config.starterKits.solutions.ctaButton}
+            </PrimaryButton>
+          </div>
+        </section>
+        <section
+          aria-label="Example Roadmap"
```

Comment:
[dust] Same comment as above.

Reply:
[sand] I wonder if it would be better to separate this section out to a separate component, given that it behaves differently from the rest of the starter kit page.